frontend/src/metabase/containers/UndoListing.jsx

```diff
@@ -22,15 +23,6 @@ import {
   UndoList,
 } from "./UndoListing.styled";
 
-const mapStateToProps = (state, props) => ({
```

Comment:
Removing `mapStateToProps` and `mapDispatchToProps` in favor of using the `useSelector` and `useDispatch` hooks, see below in component body.